admin/tests-integration/create-user.sql

```diff
@@ -0,0 +1,10 @@
+-- create db
```

Comment:
It is worth marking files as `.ddl.sql` and `.dml.sql` - do we want to keep them separate?

Reply:
one file is a transaction, and naming may start to get over complicated if both types appear in one file.  

I am comfortable with it as-is

Reply:
I'm not sure I was clear.

If we need to migrate a `null` col to `not null` then we (usually) end up with two transactions:

1. Data manipulation to ensure that there is a value present in the db col (INSERT, UPDATE)
2. Data definition change where we run ALTER TABLE

In my mind this would appear as two files, perhaps with a naming scheme that identified them as belonging together.

We can run it all together in one file, but that seems messier to me.

Reply:
I get what you initially meant :)

given your example, I would have them in one file, as an atomic operation.

Happy to go with the wider vote if this makes things clearer for everyone

Reply:
✅ then my question is answered